models/bgrid_solo/work/workshop_setup.sh

```diff
@@ -39,12 +39,18 @@
 #----------------------------------------------------------------------
 
 # Build the DART programs 
-echo 'runing quickbuild.sh'
+echo 'running quickbuild.sh'
 ./quickbuild.sh nompi
```

Comment:
why not error out if quickbuild.sh fails? Some of the other commands error out

./perfect_model_obs || exit 41
./filter || exit 51

or why not `set -e` to exit the script on any error?